python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py

```diff
@@ -186,6 +188,33 @@ def test_old_enkf_scaling_job(setup_ert):
 
     assert_obs_vector(obs_vector, 3.1622776601683795)
 
+def test_new_enkf_scaling_job(setup_ert):
+    res_config = ResConfig(config=setup_ert)
+    ert = EnKFMain(res_config)
+
+    obs = ert.getObservations()
+    obs_vector = obs["POLY_OBS"]
+
+    assert_obs_vector(obs_vector, 1.0)
+
+    job_config = {"CALCULATE_KEYS": {"keys": ["POLY_OBS"]}}
+
+    with open("job_config.yml", "w") as fout:
+        yaml.dump(job_config, fout)
+
+    job = ert.getWorkflowList().getJob("CORRELATE_OBSERVATIONS")
```

Comment:
More suggestions shall come:
OUTLIER_OBS_SCALING
PCA_OBS_SCALING

Reply:
Good suggestions, would like it to be as simple and descriptive as possible. The question is, if we expand the usage in the future, should we make a new job, or a more complex config? A more complex config where the user can for example specify type of normalization, scaling factor calculation, etc.